devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -200,15 +201,19 @@ def lowerFanSpeed() {
 }
 
 def low() {
-	setLevel(32)
+	setLevel(100 / maxSupportedSpeeds)
```

Comment:
Given the comment that I had left in this file some revisions ago, `// The GE, Honeywell, and Leviton treat 33 as medium, so account for that`, this logic is going to be troublesome.

Reply:
@dkirker Right, I see the problem. Maybe I will go for something similar  like in the 1st commit so -> setLevel(has4Speeds() ? 25 : 32)